src/commands/editContainer/editContainer.ts

```diff
@@ -48,9 +52,42 @@ export async function editContainer(context: IActionContext, node?: ContainersIt
         managedEnvironment: await getManagedEnvironmentFromContainerApp({ ...context, ...subscriptionContext }, containerApp),
         containerApp,
         containersIdx,
+        isDraftCommand: true,
     };
     wizardContext.telemetry.properties.revisionMode = containerApp.revisionsMode;
 
+    if (isAzdExtensionInstalled()) {
+        wizardContext.telemetry.properties.isAzdExtensionInstalled = 'true';
+    }
+
+    const resourceGroups: ResourceGroup[] = await ResourceGroupListStep.getResourceGroups(wizardContext);
+    wizardContext.resourceGroup = nonNullValue(
+        resourceGroups.find(rg => rg.name === item.containerApp.resourceGroup),
+        localize('containerAppResourceGroup', 'Expected to find the container app\'s resource group.'),
+    );
+
+    // Log resource group
+    wizardContext.activityChildren?.push(
```

Comment:
I feel like it might be time to make a utility function to just generate all of this boilerplate for you. 

If not a util function, maybe include adding these activity children when you call `pickContainerApp` or `pickManagedEnvironment`? Since that seems to be where you are setting all of the wizard's contexts.

I feel like just adding them into the prompt steps themselves would be the best way to do it, but since these are top-level Azure resources, that might not be possible?

Reply:
Oh interesting, I hadn't thought about doing it that way, but that might be a clean way to handle this boilerplate.  I'll investigate that as a potential solution.

Reply:
I consolidated most of the boilerplate under a new starting resource log step, let me know if you like that implementation or think it should be encapsulated differently!